backend/src/game/dice.py

```diff
@@ -0,0 +1,28 @@
+"""
+This is a dice module for the game King of Tokyo
+
+The dice can have the following face values:
+    - 1 - 3: These are the amount of victory points earned
+    - attack: Depending on the player's location, hurts either monsters
+    in or outside Tokyo
+    - heal: Heals the player up to a maximum of 10 points
+    - energy: Gives the player an energy cube to spend on upgrade cards
+"""
+
+import random
+
+DICE_VALUES = ["1", "2", "3", "attack", "heal", "energy"]
+
+
+def roll() -> str:
+    """
+    Returns back a randomly selected dice value.
+    """
```

Comment:
In the sample pull request from @borisv13 he states, "4. No comments. Good code does not require comments unless you're doing some super sophisticated algorithm. Good naming is key." I was under the impression that means as a rule we would try to avoid comments like this

Reply:
I vehemently disagree regarding the commenting, especially with the code sample that was provided in Player.py.

What was given looked much more like Java code, especially with the use of enums and constants, versus something that would be more obvious on its face.

Comments should always be included, especially docstrings, which is a Pythonic standard.

Reply:
copying reply from @borisv13  into conversation to resolve conversation:

> From the Google python style guide: A function must have a docstring, unless it meets all of the following criteria: - not externally visible - very short - obvious I stand corrected.